index_en.html

```diff
@@ -424,6 +418,27 @@ <h2 class="clearfix">
 
 		</section>
 
+        <section class="sectionTicket" id="tickets">
+            <header class="sectionHeadArea">
+                <h2 class="clearfix">
+                    <span class="ttlSubLCenter">Ticket</span>
+                    <span class="ttlSubLLeft"><img src="img/common/ttl_dots_left.png"></span>
+                    <span class="ttlSubLRight"><img src="img/common/ttl_dots_right.png"></span>
+                </h2>
+            </header>
+            <div class="sectionTicketInner">
+                <p>
+                You need a ticket to participate.
```

Comment:
Registration fee is required to participate the conference.

Please purchase the tickets from Doorkeeper.